src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs

```diff
@@ -6,6 +6,9 @@ public sealed class ChocolateyPackage : Resource
     {
         public string Package { get; set; }
         public PackageState Ensure { get; set; }
+        public bool? PreRelease { get; set; }
+        public bool? IgnoreChecksum { get; set; }
```

Comment:
The booleans do not have to be nullable here.

Reply:
🤦‍♂️ I erred on the side of caution 🤣
I'll fix tomorrow.